docs/internal-documentation/templates/specifications/tests/Template01.md

```diff
@@ -65,7 +65,7 @@ giving a correct DNS response for an authoritative name server.
 
 ## Summary
 
-> > First we can have bullets, if applicable, that states noteable things about
+> > First we can have bullets, if applicable, that states notable things about
```

Comment:
I think "that states" should be "that state" as it refers to the bullets.

Reply:
Fixed.